### DISCUSSION TITLE
Search feature with internal server error

### DISCUSSION THREAD

**Discussion by alexisgec** [rtenorioh/Press-Ticket#96] [General]
Hi! can you help me with this? in search i have an Internal Server Error, not sure how to debug it..

https://prnt.sc/XzYHnGKH5z2I
https://prnt.sc/YjFpNcWkUOQR

do you have some advice?

**Comment by rtenorioh** [comment]
I believe this error has already been fixed, if it still occurs, please report it again.

**Comment by rtenorioh** [comment]
If the error persists, please contact us again.
